meta-lxatac-bsp/recipes-bsp/barebox/barebox_2023.10.0.bb

```diff
@@ -6,10 +6,12 @@ SRC_URI += " \
 "
 require files/patches/series.inc
 
-SRC_URI[sha256sum] = "d0f78a69ba240327247c8fd0e1d45287e4a0dff99ed847e9a696cc2da0cf388c"
+SRC_URI[sha256sum] = "07823aa3cdc083087788a1cccded99a15a250746e259c2ff905dc48f8d9f409a"
 
 COMPATIBLE_MACHINE = "lxatac"
 
+BAREBOX_IMAGES = "*.stm32"
```

Comment:
It would be nice if we could import the [barebox recipe that eventually should make it into OE-core](https://lore.kernel.org/all/20230425184720.456896-1-ejo@pengutronix.de/) in a future update. This parses the `barebox-flash-images` file, so hardcoding `BAREBOX_IMAGES` as done here is unneeded.

Reply:
I've done so now in a separate commit. Please have a look at it.

Reply:
LGTM. Thanks for giving this some wider testing!